src/ICeWall/doc/add_pad.md

```diff
@@ -0,0 +1,51 @@
+## add_pad
+
+### Synopsis
+```
+  % add_pad [-name <name>] \
+                    [-type <type>] \
+                    [-cell <library_cell>] \
+                    [-signal <signal_name>] \
+                    [-edge (bottom|right|top|left)] \
+                    [-location {(centre|origin) {x <value> y <value>} [orient (R0|R90|R180|R270|MX|MY|MXR90|MYR90)]}] \
+                    [-bump {row <number> col <number>}] \
```

Comment:
There seems to be no error checking still with respect to the bump row and column and the x/y placement of the pad. Also setting the bump number to greater and the number of rows or columns or setting it to 0 or -1, does not generate an error, but it probably should.

Reply:
Checks added